benjaminflodin/tic_tac_arrows.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <conio.h>
+
+// class gameBoard{
+// public:
+
+
+
+// };
+
+
+char game_board[3][9] = {                           // index (0,2), (4,6), (8,10) == [] // index 1 4 7 == spot for player marker
```

Comment:
varför 3x9 inte 3x3 eller en plan 9 array ?

Reply:
Jag läste fel först och fortsatte på samma spår så det blev nog krångligare än det hade behövt vara :)

Reply:
vg uppdatera programmet och skicka in en ny commit

Reply:
Jag skrev om hela programmet, har laddat upp ny version nu